Source/engine/render/automap_render.cpp

```diff
@@ -22,12 +22,15 @@ template <DirectionX DirX, DirectionY DirY>
 void DrawMapLine(const CelOutputBuffer &out, Point from, int height, std::uint8_t colorIndex)
 {
 	while (height-- > 0) {
+		out.SetPixel({ from.x, from.y + 1 }, 0);

```

Comment:
This can also be written as:
```suggestion
		out.SetPixel(from + DIR_S, 0);
```

Reply:
even better, was looking for the enum but couldn't find : P

Reply:
{ 0, 1 } is `DIR_SW`, not `DIR_S`.

Reply:
Probably not appropriate here.